install_scripts/templates/common/docker-version.sh

```diff
@@ -157,30 +157,31 @@ getMaxDockerVersion() {
         fi
     fi
     if [ "$LSB_DIST" = "ubuntu" ]; then
-        # Max Docker version on Ubuntu 14.04 is 18.06.1.
+        # Max Docker version on Ubuntu 14.04 is 18.06.2.
         if [ "$DIST_VERSION" = "14.04" ]; then
-            MAX_DOCKER_VERSION_RESULT="18.06.1"
+            MAX_DOCKER_VERSION_RESULT="18.06.2"
         fi
     fi
     if [ "$LSB_DIST" = "debian" ]; then
         # Max Docker version on Debian 7 is 18.03.1
         if [ "$DIST_VERSION" = "7" ]; then
             MAX_DOCKER_VERSION_RESULT="18.03.1"
         fi
-        # Max Docker version on Debian 8 is 18.06.1.
+        # Max Docker version on Debian 8 is 18.06.2.
         if [ "$DIST_VERSION" = "8" ]; then
-            MAX_DOCKER_VERSION_RESULT="18.06.1"
+            MAX_DOCKER_VERSION_RESULT="18.06.2"
         fi
     fi
     # 2019-01-07
     # Max Docker version on Amazon Linux 2 is 18.06.1.
+    # NOTE: need to patch here with 18.06.2 when available.
     if [ "$LSB_DIST" = "amzn" ]; then
         MAX_DOCKER_VERSION_RESULT="18.06.1"
```

Comment:
Unclear whether this is usable, but this Amazon bulletin suggests that a  docker-18.06.1ce-7.25 patch is available.

https://alas.aws.amazon.com/ALAS-2019-1156.html